ui/src/pages/insightDetail/components/eventAggregator/styles.module.less

```diff
@@ -380,261 +376,278 @@
-              border-radius: 6px;
-              padding: 12px;
-              margin: 12px 0;
-              overflow-x: auto;
+              h4 {
+                font-size: 16px;
+              }
 
-              code {
-                background: transparent;
-                padding: 0;
-                color: #f0e6ff;
+              h5 {
+                font-size: 14px;
               }
-            }
 
-            code {
-              background: rgba(147, 112, 219, 0.2);
-              padding: 2px 6px;
-              border-radius: 4px;
-              color: #f0e6ff;
-              font-family: 'Menlo', 'Monaco', 'Courier New', monospace;
-            }
+              h6 {
+                font-size: 13px;
+              }
 
-            ul,
-            ol {
-              padding-left: 24px;
-              margin: 8px 0;
+              pre {
+                background: rgba(0, 0, 0, 0.4);
+                border: 1px solid rgba(147, 112, 219, 0.5);
+                border-radius: 6px;
+                padding: 12px;
+                margin: 12px 0;
+                overflow-x: auto;
 
-              li {
-                margin: 4px 0;
+                code {
+                  background: transparent;
+                  padding: 0;
+                  color: #f0e6ff;
+                }
               }
-            }
 
-            blockquote {
-              color: rgba(230, 230, 250, 0.85);
-              border-left: 4px solid rgba(147, 112, 219, 0.4);
-              background: rgba(147, 112, 219, 0.1);
-              margin: 16px 0;
-              padding: 12px 16px;
-              border-radius: 0 8px 8px 0;
-            }
+              code {
+                background: rgba(147, 112, 219, 0.2);
+                padding: 2px 6px;
+                border-radius: 4px;
+                color: #f0e6ff;
+                font-family: 'Menlo', 'Monaco', 'Courier New', monospace;
+              }
 
-            a {
-              color: #9370db;
-              text-decoration: none;
-              transition: all 0.2s ease;
-              border-bottom: 1px solid transparent;
+              ul,
+              ol {
+                padding-left: 24px;
+                margin: 8px 0;
 
-              &:hover {
-                color: lighten(#9370db, 10%);
-                border-bottom-color: currentColor;
+                li {
+                  margin: 4px 0;
+                }
               }
-            }
 
-            table {
-              border-collapse: separate;
-              border-spacing: 0;
-              width: 100%;
-              margin: 16px 0;
-              border-radius: 8px;
-              border: 1px solid rgba(147, 112, 219, 0.2);
-              overflow: hidden;
-
-              th,
-              td {
-                border: 1px solid rgba(147, 112, 219, 0.2);
-                padding: 12px;
+              blockquote {
+                color: rgba(230, 230, 250, 0.85);
```

Comment:
The .events_content_diagnosisLoading class could use a more specific naming convention to better reflect its state.